korman/nodes/node_conditions.py

```diff
@@ -417,22 +417,22 @@ def export(self, exporter, bo, parent_so):
         enter_simple = self.find_input_socket("enter").allow
         enter_settings = self.find_input("enter", "PlasmaVolumeReportNode")
         if enter_simple or enter_settings is not None:
-            key = self._export_volume_event(exporter, region_bo, region_so, plVolumeSensorConditionalObject.kTypeEnter, enter_settings)
+            key = self._export_volume_event(exporter, region_bo, region_so, parent_so, plVolumeSensorConditionalObject.kTypeEnter, enter_settings)
             interface.addIntfKey(key)
 
         # Region Exits
         exit_simple = self.find_input_socket("exit").allow
         exit_settings = self.find_input("exit", "PlasmaVolumeReportNode")
         if exit_simple or exit_settings is not None:
-            key = self._export_volume_event(exporter, region_bo, region_so, plVolumeSensorConditionalObject.kTypeExit, exit_settings)
+            key = self._export_volume_event(exporter, region_bo, region_so, parent_so, plVolumeSensorConditionalObject.kTypeExit, exit_settings)
             interface.addIntfKey(key)
 
         # Don't forget to export the physical object itself!
         exporter.physics.generate_physical(region_bo, region_so, bounds=self.bounds,
                                            member_group="kGroupDetector",
                                            report_groups=self.report_on)
 
-    def _export_volume_event(self, exporter, bo, so, event, settings):
+    def _export_volume_event(self, exporter, bo, so, parent_so, event, settings):
```

Comment:
Now that we are juggling two different scene objects, I think we should rename `bo` and `so` to be more descriptive.